### PR TITLE
inadyn: addon update to 2.7 and gettext fix 

### PR DESCRIPTION
**Pull request by heitbaum**
Update to current inadyn
- 18 month bump, but limited changes
- https://github.com/troglobit/inadyn/blob/master/ChangeLog.md

update libconfuse (dependency of inadyn)
- 2 year bump - minimal changes
- Includes get text 0.20 patch
- https://github.com/libconfuse/libconfuse/blob/master/ChangeLog.md

### CODE REVIEW

File: packages/addons/addon-depends/inadyn/libconfuse/patches/gettext-0.20-libconfuse.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- libconfuse-3.3/configure.ac	2020-06-25 05:21:59.000000000 +0000
++++ libconfuse-3.3/configure.ac	2020-12-29 02:04:36.074750823 +0000
+@@ -29,7 +29,7 @@
+ # Unfortunately CentOS7, RHEL7 ships 0.18.2.1, so for best compat.
+ # level at this point in time we set 0.18.2.
+ AM_GNU_GETTEXT([external])
+-AM_GNU_GETTEXT_VERSION([0.18.2])
++AM_GNU_GETTEXT_VERSION([0.20])
```
Comment (CvH) [inline packages/addons/addon-depends/inadyn/libconfuse/patches/gettext-0.20-libconfuse.patch:8]: can we do it that way
https://github.com/LibreELEC/LibreELEC.tv/pull/4833/files
that would be upstream able too

File: packages/addons/addon-depends/inadyn/libconfuse/package.mk [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 
 PKG_NAME="libconfuse"
-PKG_VERSION="3.2.2"
-PKG_SHA256="2cf7e032980aff8f488efba61510dc3fb95e9a4b9183f985dea457a5651b0e2c"
-PKG_LICENSE="https://github.com/martinh/libconfuse/blob/master/LICENSE"
-PKG_SITE="https://github.com/martinh/libconfuse"
-PKG_URL="https://github.com/martinh/libconfuse/archive/v$PKG_VERSION.tar.gz"
+PKG_VERSION="3.3"
+PKG_SHA256="cb90c06f2dbec971792af576d5b9a382fb3c4ca2b1deea55ea262b403f4e641e"
+PKG_LICENSE="https://github.com/libconfuse/libconfuse/blob/master/LICENSE"
```
Comment (CvH) [inline packages/addons/addon-depends/inadyn/libconfuse/package.mk:7]:
```suggestion
PKG_LICENSE="ISC"
```

Comment (CvH) [reply packages/addons/addon-depends/inadyn/libconfuse/package.mk:7]: needs https://github.com/spdx/license-list-data/blob/master/text/ISC.txt copied to `LE/licenses/`

File: packages/addons/addon-depends/inadyn/libconfuse/patches/gettext-0.20-libconfuse.patch (hunk 1 above)
Comment (heitbaum) [reply packages/addons/addon-depends/inadyn/libconfuse/patches/gettext-0.20-libconfuse.patch]: Patch updated.

File: packages/addons/addon-depends/inadyn/libconfuse/package.mk (hunk 2 above)
Comment (heitbaum) [reply packages/addons/addon-depends/inadyn/libconfuse/package.mk]: License updated

**Comment by heitbaum** [comment]
Note: PR raised on upstream: https://github.com/libconfuse/libconfuse/pull/152

File: licenses/ISC.txt [hunk 3]
```diff
@@ -0,0 +1,1041 @@
+
```
Comment (CvH) [inline licenses/ISC.txt:1]: you pulled the html instead the plain txt

File: packages/addons/addon-depends/inadyn/libconfuse/package.mk (hunk 2 above)
Comment (heitbaum) [reply packages/addons/addon-depends/inadyn/libconfuse/package.mk]: Note to self: no rushing to fix PR before heading the the office half 😴. Fixed.

**Review by CvH — APPROVED** [review]

**Pull request merged**